cip-constitution-tech/README.md

```diff
@@ -0,0 +1,255 @@
+
+### Safety
+
+Without describing best practices for the form and handling of the constitution, we risk the constitution document being stored in an insecure manner.
+By storing the constitution on a decentralized platform, we can ensure it's immutability and permissionless access.
+This is a step to improve the longevity and accessibility of each constitution iteration.
+
+### Interoperability
+
+By defining a file extension and formatting rules for the constitution we ensure that tooling reading and writing the constitution will be interoperable.
+Furthermore we aim to make the role of constitution iteration comparison tools easier, by minimizing formatting and style changes between iterations.
+This will reduce compatibility issues between tools, promoting the accessibility of the constitution.
+
+### Usability
+
+Rich text formatting greatly enhances the readability of text, especially in large complex documents.
+Without the ability to format text, it could easily become cumbersome to read, negatively effecting the accessability of the Cardano constitution.
+
+## Specification
+
+The following specification SHOULD be applied to all constitution iterations.
+This standard could be augmented in the future via a separate CIP which aims to replace this one.
+
+### Sentences
+
+The constitution text MUST only contain at maximum one sentence per line.
+Each sentence MUST be followed by a newline.
+
+Each sentence SHOULD start on its own line with a capitalized letter.
+
+Example:
+
+```md
+This is a short sentence on one line.
+
+This is a long sentence and I have valid reasons for it being so long,
+such as being an example of a long sentence.
+```
+
+It is recommended that each line is ended by either a full stop or comma.
+
+### File Type
+
+The constitution file MUST be a text file named `constitution.txt`.
+
+### Hashing
+
+When supplying a constitution hash digest to chain, the algorithm used MUST be Blake2b-256.
+Before creating a hash digest the constitution plain text MUST be in its raw text, including any [Rich Text Formatting](#rich-text-formatting) related characters.
+
+### Storage
+
+The each ratified constitution MUST be stored, immutably on a distributed storage mechanism such as IPFS.
+Where backups can be easily made in a permissionless manner by interested parties.
+This storage platform MUST be easily accessible, with strong tooling support.
+Authors SHOULD NOT specify the use of centralized gateways to access the constitution plain text.
+
+// only ascii
+
+### Rich Text Formatting
+
+The constitution text MAY include a strict subset of rich text styling as defined in this specification.
+Tooling rendering the constitution SHOULD recognize these and render them faithfully.
+
+#### Headers
+
+Headers are denoted via a hashtag character `#` followed by a space ` `.
+There are six levels to headers with, each being defined via a extra `#`.
```

Comment:
```suggestion
There are six levels of headers, with each lower level set via an added `#`.
```

Reply:
There is probably a standard way of describing this which can be cobbled from Markdown and/or W3C documentation.  I proposed the above since I thought it made the statement more accurate with a minimum amount of change.

Reply:
Just checked the [Markdown Guide](https://www.markdownguide.org/basic-syntax/#headings) and [GitHub Docs](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#headings) and neither contains a description that's both concise and complete, so the above is the best I can offer. 😖